projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/container/multi-step-checkout.component.ts

```diff
@@ -215,10 +215,60 @@ export class MultiStepCheckoutComponent implements OnInit, OnDestroy {
     this.tAndCToggler = !this.tAndCToggler;
   }
 
+  initializeCheckoutNavBar(): CheckoutNavBarItem[] {
```

Comment:
shouldn't we keep this data in the store?

Reply:
After discussion with @kacperknapik we leave it here as it is for now